Uno.Gallery/Platforms/WebAssembly/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "declaration": true,
     "diagnostics": true,
     "noImplicitAny": true,
-    "outFile": "WasmScripts/UnoGallery.js",
+    //"outFile": "WasmScripts/UnoGallery.js",
```

Comment:
Is the file still generated? It may cause other kinds of issues

Reply:
@jeromelaban It's generated in OutputPath same as we do in Uno. It causes issues otherwise when the compiled `.d.ts` are present and then passed again as input